src/go/rpk/pkg/cli/cmd/redpanda/config.go

```diff
@@ -147,6 +147,14 @@ func bootstrap(fs afero.Fs) *cobra.Command {
 			cfg.Redpanda.SeedServers = []config.SeedServer{}
 			cfg.Redpanda.SeedServers = seeds
 
+			// The Redpanda binary defaults to the root-driven style of cluster
+			// formation. In bootstrapping a new cluster, use seeds-driven
```

Comment:
Are we expecting users to not run `rpk redpanda config bootstrap` in the future? AFAICT it's our currently recommended way to create a base config file.

Reply:
Yes, we want to continue using `config bootstrap` as it's the recommended way to deploy, but we want to have it configure the new style of bootstrapping. Would removing the first sentence here make this less confusing?